PatternPal/PatternPal.Extension/Commands/SubscribeEvents.cs

```diff
@@ -451,16 +480,47 @@ private static void OnDocumentSaved(Document document)
             IList<RecognizeResult> recognizeResults)
         {
             ThreadHelper.ThrowIfNotOnUIThread();
-            if (_package == null || !Privacy.Instance.DoLogData)
+
+            if (_package == null || !Privacy.Instance.DoLogData 
```

Comment:
I thought this was fixed, but shouldn't this be DoLog instead of the settings instance?